src/python/pants/option/arg_splitter.py

```diff
@@ -194,15 +194,15 @@ def assign_flag_to_scope(flg: str, default_scope: str) -> None:
     if not goals and not self._help_request:
       self._help_request = NoGoalHelp()
 
-    return SplitArgs(list(goals), scope_to_flags, positional_args, passthru,
+    return SplitArgs(list(goals), scope_to_flags, specs, passthru,
                      passthru_owner if passthru else None, self._unknown_scopes)
 
   @staticmethod
-  def is_positional_arg(arg: str) -> bool:
-    """Does arg 'look like' a positional arg (rather than a goal name).
+  def spec(arg: str) -> bool:
+    """Does arg 'look like' a spec (rather than a goal name).
```

Comment:
Nit: might be worth capitalizing or referring to the defined interface from docs.